pkg/printers/internalversion/describe.go

```diff
@@ -1321,6 +1321,14 @@ func describeContainerBasicInfo(container api.Container, status api.ContainerSta
 			w.Write(LEVEL_2, "Port:\t%s\n", portString)
 		}
 	}
+	hostPortString := describeContainerHostPorts(container.Ports)
+	if strings.Contains(hostPortString, ",") {
+		w.Write(LEVEL_2, "HostPorts:\t%s\n", hostPortString)
+	} else {
+		if len(hostPortString) == 1 {
```

Comment:
This condition looks weird. When we the length of the string can be 1?

Reply:
My mistake. Updated. PTAL. Thanks.